src/init.cpp

```diff
@@ -1433,6 +1428,11 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     };
     ApplyArgsManOptions(args, mempool_opts);
     mempool_opts.check_ratio = std::clamp<int>(mempool_opts.check_ratio, 0, 1000000);
+
+    int64_t descendant_limit_bytes = mempool_opts.limits.descendant_size_vbytes * 40;
```

Comment:
nit in 470347d2ffd462d635fdf344e38332a4eb4b8ad3: could add a `{}` scope to avoid leaking this symbol into the full `AppInitMain` scope?

Reply:
I'm not going to do this in this PR, feel free to follow up.